README.md

```diff
@@ -158,23 +178,41 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | gwip         | commit a work-in-progress branch                     |
 | gunwip       | uncommit the work-in-progress branch                 |
 
+### Tags
+
+| Abbreviation | Command                                              |
+| ------------ | ---------------------------------------------------- |
+| gts          | `git tag -s`                                         |
+| gtv          | `git tag | sort -V`                                  |
+| gtl          | list tags matching prefix                            |
+
+
+### Files
+
+| Abbreviation | Command                                              |
+| ------------ | ---------------------------------------------------- |
+| ga           | `git add`                                            |
+| gaa          | `git add --all`                                      |
+| gapa         | `git add --patch`                                    |
+| grm          | `git rm`                                             |
+| grmc         | `git rm --cached`                                    |
+| grs          | `git restore`                                        |
+| grss         | `git restore --source`                               |
+
+
 ### Everything Else
 
 | Abbreviation | Command                                                     |
 | ------------ | --------------------------------------------------------    |
 | g            | `git`                                                       |
-| ga           | `git add`                                                   |
-| gaa          | `git add --all`                                             |
-| gapa         | `git add --patch`                                           |
```

Comment:
I personnaly use this shortcut a lot! Please don't remove it or explain why you want to remove it!

Reply:
Oups sorry: didn't see you just put it in a specific section.

Reply:
Yeah, sorry the changes were a bit fragmented, so it's a bit hard to follow. I didn't remove or change any semantics of the defined abbreviations or functions in this PR (except `gpu` - which `--set-upstream` goes after the branch name - which is okay for most cases).